src/url-replacements.js

```diff
@@ -239,6 +239,14 @@ class UrlReplacements {
       return this.getTimingData_('navigationStart',
           'domContentLoadedEventStart');
     });
+
+    // Returns an identifier for the viewer.
+    this.set_('VIEWER', () => {
+      return viewerFor(this.win_).getViewerOrigin().then(origin => {
+        const url = parseUrl(origin);
```

Comment:
Origin can be empty, so let's handle here. Otherwise it will resolve relative URL against the empty string.